Libraries/LibJS/Runtime/AsyncFunctionDriverWrapper.cpp

```diff
@@ -20,8 +20,7 @@ JS_DEFINE_ALLOCATOR(AsyncFunctionDriverWrapper);
 NonnullGCPtr<Promise> AsyncFunctionDriverWrapper::create(Realm& realm, GeneratorObject* generator_object)
 {
     auto top_level_promise = Promise::create(realm);
-    // Note: This generates a handle to itself, which it clears upon completing its execution
-    //       The top_level_promise is also kept alive by this Wrapper
+    // Note: The top_level_promise is also kept alive by this Wrapper
```

Comment:
Git blame me, IIRC....
But wow 9Gig of promises seems a bit excessive...

Reply:
It’s not just the promises themselves, they keep the entire world alive since they reference the realm they’re in 😅